Makefile

```diff
@@ -12,7 +12,7 @@ WITH_GOFLAGS = GOFLAGS=$(GOFLAGS)
 WITH_RELEASE_REPO = KO_DOCKER_REPO=$(RELEASE_REPO)
 
 ## Extra helm options
-CLUSTER_NAME ?= $(shell kubectl config view --minify -o jsonpath='{.clusters[].name}' | rev | cut -d"/" -f1 | rev)
+CLUSTER_NAME ?= $(shell kubectl config view --minify -o jsonpath='{.clusters[].name}' | rev | cut -d"/" -f1 | rev | cut -d"." -f1)
```

Comment:
I think this will work for EKS but breaks kops since Kops uses the FQDN

Reply:
Yes I realized that we need a more consistent and inclusive way to config the Karpenter deployment process instead of hacking the environment variable.  Created an issue #1075 to track this.

Reply:
For kOps the true way is `kubectl config current-context`